MDANSE/Src/MDANSE/Framework/Jobs/IJob.py

```diff
@@ -276,12 +276,11 @@ def process_tasks_queue(self, tasks, outputs, log_queues):
         return True
 
     def _run_multicore(self):
+        if hasattr(self._status, "_queue_0"):
+            self._status._queue_0.put("started")
 
-        ctx = multiprocessing.get_context("spawn")
-
-        manager = ctx.Manager()
-        inputQueue = manager.Queue()
-        outputQueue = manager.Queue()
+        inputQueue = Queue()
+        outputQueue = Queue()
```

Comment:
On my computer, this change ensures that all child processes were terminated when requested from GUI. With the code before it there always seemed to be one last process left that was still running.

However, the new code causes issues with macos as Queue.qsize() is not implemented for some reason. The reimplemented Queue is added as suggested by on of the answers in https://stackoverflow.com/questions/65609529/python-multiprocessing-queue-notimplementederror-macos.